packages/courier-js/src/index.ts

```diff
@@ -15,7 +19,11 @@ const client = {
     if (!userId) {
       throw new Error("userId is required");
     }
-    await this.instance.post(`identify/${userId}`, payload);
+    await this.instance.post(`identify/${userId}`, {
+      profile: {
+        ...payload,
+      },
+    });
```

Comment:
Why this change? This seems odd to include

Reply:
I could open a separate PR for this. This was a regression introduced while changing `PUT` to `POST`